config/config.go

```diff
@@ -489,6 +499,11 @@ func getChannelForConsumer(consumerSection *ini.Section, seedChannels []SeedChan
 	return channel, sql.ErrNoRows
 }
 
+func getConsumerType(consumerSection *ini.Section) (ConsumerType, error) {
+	consumerTypeString := consumerSection.Key("type").MustString("")
```

Comment:
Should we make the `type` field optional (with "push" being the default)?
Otherwise all existing configs will need to be updated, or they would break